website/src/components/APITable/index.tsx

```diff
@@ -1,27 +1,27 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
 import React, {
   type ComponentProps,
   type ReactElement,
   type ReactNode,
   isValidElement,
   useRef,
   useEffect,
+  useCallback,
+  memo,
 } from 'react';
 import useBrokenLinks from '@docusaurus/useBrokenLinks';
-import {useHistory} from '@docusaurus/router';
+import { useHistory } from '@docusaurus/router';
 import styles from './styles.module.css';
 
 interface Props {
   readonly children: ReactElement<ComponentProps<'table'>>;
   readonly name?: string;
 }
 
+// Utility function to sanitize strings (prevent injection attacks)
+function sanitizeString(str: string): string {
+  return str.replace(/</g, '&lt;').replace(/>/g, '&gt;');
+}
```

Comment:
There's no reason to sanitize inputs under our control. Why do you want to prevent HTML injection exactly?